### PR TITLE
Bug 1455697 - provide newTaskId in context, too

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Comment by djmitche** [comment]
Sorry @jonasfj I'm going to need this soon :)

File: docs/firing-hooks.md [hunk 1]
```diff
@@ -17,9 +17,10 @@ submits it to the Queue service via `Queue.createTask`.
 A hook definition's `task` property is, in fact, a [JSON-e](https://taskcluster.github.io/json-e/) template.
 When a hook is fired, that template is rendered and the result is submitted to `Queue.createTask`.
 
-The context for that rendering is an object with property `firedBy`, giving
-the action that led to the hook firing. The other properties of the object vary
-depending on this property.
+The context for that rendering is an object with property `firedBy`, giving the
+action that led to the hook firing; and `newTaskId` giving the taskId of the
```
Comment (jonasfj) [inline docs/firing-hooks.md]: Can we call it `taskId` or `ownTaskId`

Comment (jonasfj) [reply docs/firing-hooks.md]: Also what about `taskGroupId`

File: docs/firing-hooks.md [hunk 2]
```diff
@@ -17,9 +17,10 @@ submits it to the Queue service via `Queue.createTask`.
...
+task being created. The other properties of the object vary depending on the
+`firedBy` property.
```
Comment (jonasfj) [inline docs/firing-hooks.md]: Please document and ensure that properties in the context can overwrite the built-in context properties... This way we can add more built-in properties without breaking compatibility..

File: docs/firing-hooks.md (hunk 1 above)
Comment (djmitche) [reply docs/firing-hooks.md]: I'll rename it.  The taskGroupId = taskId so there's no need to specify it separately.  I'll add that to the doc, though.

**Pull request merged**